src/native/public/mono/metadata/object-forward.h

```diff
@@ -19,4 +19,9 @@ typedef struct _MonoException MONO_RT_MANAGED_ATTR MonoException;
 typedef struct _MonoReflectionAssembly MONO_RT_MANAGED_ATTR MonoReflectionAssembly;
 typedef struct _MonoReflectionTypeBuilder MONO_RT_MANAGED_ATTR MonoReflectionTypeBuilder;
 
+typedef struct _MonoVarianceSearchTableEntry {
+    int depth, offset;
+    MonoClass *klass;
+} MonoVarianceSearchTableEntry;
```

Comment:
Don't put it in the public header.  `class-internals.h` is fine

Reply:
Also `object-forward.h` is mean for forward declarations only - for code that just needs opaque pointers to mono structs, without knowing the members